gef.py

```diff
@@ -4757,6 +4761,62 @@ def __set_repeat_count(self, argv: List[str], from_tty: bool) -> None:
         return
 
 
+@register
+class PieCommand(GenericCommand):
+    """Manage the current loaded architecture"""
```

Comment:
nit: Period at the end for rendered docs.

Reply:
Done